app/models/framework/sheet.rb

```diff
@@ -7,6 +7,12 @@ def export_mappings
         @export_mappings ||= {}
       end
 
+      ##
+      # E.g. 'Total Cost (ex VAT)'
+      def total_value_field(value = nil)
```

Comment:
/s/value/field_name?

Reply:
Probably better, yes, though has to happen in lots of places and am futzing with the backfill query

Reply:
Hm I've gone with what's on [the card](https://trello.com/c/i05471TJ/415-submission-entries-have-an-explicit-total-value-attribute-column) here – trying to keep the new column name and this lined up

Reply:
Ah my bad, I wasn’t specific enough, I meant the parameter name rather than the method name. Totally tweaking at the margins but thought it would aid comprehension by giving more of a signal as to the thing being passed in.

Reply:
Ah yeah there is a bit of overloading going on here. I'm trying to be consistent with the other DSL-style [methods](https://github.com/dxw/DataSubmissionServiceAPI/blob/415-submission-entries-have-an-explicit-total-value-attribute-column/app/models/framework/definition.rb#L18-L26) in `Definition` – in these methods the getter and the setter is combined such that you can dereference with `my_sheet.total_value_field` or set with `total_value_field 'The new value'` in the DSL. Perhaps `new_value` would work? Otherwise it'll end up looking like `def total_value_field(total_value_field = nil)` which I don't seem to be keen on for some reason.

Reply:
Ah I see you mean `def total_value_field(field_name = nil)`

Reply:
Not fully awake yet.